extension/content.js

```diff
@@ -26,6 +31,55 @@ function linkifyBranchRefs() {
 	});
 }
 
+function commentIsUseless(type, el) {
+	if (uselessContent[type].text.indexOf(el.innerText) >= 0) {
+		return true;
+	}
+	// check if there is exactly one child element, that has one other child node
+	// using `childNodes` because this also includes text
```

Comment:
In the future I'd like to make this more flexible, but this is good enough for a first version.

Like, would be nice to support eg, `👍Yes!` or a emoji vote + one word or similar. Or maybe if longer just remove the emoji, but not the comment.